### PR TITLE
Filter sort Landing-page

### PR DESCRIPTION
**Pull request by ark2002**
- Added Filter Functionality
- Added Moments
- Added Landing Page

### CODE REVIEW

File: src/frontend/utils/index.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export { notesLabelsFiltered } from "./notesLabelsFiltered"
```
Comment (ivinayakg) [inline src/frontend/utils/index.js:1]: good job for exporting functions like this

File: src/frontend/utils/notesLabelsFiltered.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+export const notesLabelsFiltered = (notes, filters) => {
+    switch (filters) {
+        case "LABEL 1":
+            return [...notes.filter((note) => note.label === "Label 1")];
```
Comment (ivinayakg) [inline src/frontend/utils/notesLabelsFiltered.js]: no need to spread it

File: src/frontend/utils/notesPriorityFiltered.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+export const notesPriorityFiltered = (notes, filters) => {
+    switch (filters) {
+        case "HIGH":
+            return [...notes.filter((note) => note.priority === "HIGH")];
+        case "MEDIUM":
+            return [...notes.filter((note) => note.priority === "MEDIUM")];
+        case "LOW":
```
Comment (ivinayakg) [inline src/frontend/utils/notesPriorityFiltered.js:7]: again no need to use spread operator

**Review by ivinayakg — APPROVED** [review]

**Review by MarkVed17 — APPROVED** [review]
- This works!
- Readable code
- Good to merge.

**Pull request merged**